### PR TITLE
docs: clarify how to deal with problematic releases

### PR DESCRIPTION
**Pull request by jviotti**
Signed-off-by: Juan Cruz Viotti <jviotti@openmailbox.org>

### CODE REVIEW

File: docs/MAINTAINERS.md [hunk 1]
```diff
@@ -44,3 +44,10 @@ Upgrading Electron
 
 - Upgrade the `electron-prebuilt` dependency version in `package.json` to an
 *exact version* (no `~`, `^`, etc).
+
+Dealing with a problematic release
+----------------------------------
+
+There can be times where a release is accidentally plagued with bugs. If you
+released a new version and notice the error rates are higher than normal, then
+revert the problematic release as soon as possible, until the bugs are fixed.
```
Comment (lurch) [inline docs/MAINTAINERS.md]: This is fairly obvious, surely? Probably be useful to document _how_ to "revert the problematic release" ;-)

Comment (jviotti) [inline docs/MAINTAINERS.md]: Sounds good, done!

File: docs/MAINTAINERS.md [hunk 2]
```diff
@@ -44,3 +44,20 @@ Upgrading Electron
...
+
+You can revert a version by deleting its builds from the S3 bucket and Bintray.
+
+Use the following command to delete files from S3:
+
+```sh
+aws s3api delete-object --bucket <bucket name> --key <file name>
+```
+
+The Bintray dashboard provides an easy way delete a version.
```
Comment (jhermsmeier) [inline docs/MAINTAINERS.md]: Looks like there's a "to" missing? i.e. `[...] an easy way *to* delete a version`

File: docs/MAINTAINERS.md [hunk 3]
```diff
@@ -44,3 +44,20 @@ Upgrading Electron
...
+There can be times where a release is accidentally plagued with bugs. If you
+released a new version and notice the error rates are higher than normal, then
+revert the problematic release as soon as possible, until the bugs are fixed.
+
+You can revert a version by deleting its builds from the S3 bucket and Bintray.
+
+Use the following command to delete files from S3:
+
+```sh
+aws s3api delete-object --bucket <bucket name> --key <file name>
```
Comment (jhermsmeier) [inline docs/MAINTAINERS.md]: How does one obtain bucket name & key? Is it worth adding that info as well?

Comment (lurch) [inline docs/MAINTAINERS.md]: Given that there are actually 8 separate files to delete, maybe a little helper-script would make this process easier?

File: docs/MAINTAINERS.md (hunk 2 above)
Comment (jviotti) [inline docs/MAINTAINERS.md]: Fixed!

File: docs/MAINTAINERS.md (hunk 3 above)
Comment (jviotti) [inline docs/MAINTAINERS.md]: Done! I clarified that only deleting a couple of files is enough, so I guess that prevents the need of a script.

**Pull request merged**